data/themes/darktable.css

```diff
@@ -960,31 +960,35 @@ notebook tabs,
 {
   min-height: 2em;
   min-width: 2em;
-  margin: 0;
   padding: 0.1em;
-  border: 0;
 }
 
 #modules-tabs #button-canvas
 {
   margin: 12px; /* not real pixels, this is used as a percent extra space */
 }
 
-#blending-tabs button
+#blending-tabs #dt-toggle-button
 {
   font-size: 0.8em;
   min-height: 1.55em;
   min-width: 1.55em;
   padding: 0.45em;
-  margin: 0;
-  border: 0;
+}
+
+#blending-tabs #dt-button
+{
+	font-size: 0.8em;
+	padding: 0.1em;
```

Comment:
Adding:

`margin: 0 0.5em;`

Avoid the button to be stuck on the right.

Reply:
Done thanks. I would have preferred the button to take the whole width (i.e. without spaces on its side), but it seems that increasing the width changes the size of the icon too...